data/sql/updates/pending_db_auth/rev_1620114805872279900.sql

```diff
@@ -0,0 +1,108 @@
+INSERT INTO `version_db_auth` (`sql_rev`) VALUES ('1620114805872279900');
+
+--
+-- Table structure for table `build_info`
+--
+DROP TABLE IF EXISTS `build_info`;
```

Comment:
we need to update the wiki explaining what this table is

Reply:
https://github.com/azerothcore/wiki/pull/466